packages/iris-grid/src/IrisGrid.jsx

```diff
@@ -3179,7 +3234,16 @@ export class IrisGrid extends Component {
               renderer={this.renderer}
               stateOverride={stateOverride}
               theme={theme}
-            />
+            >
+              {this.renderOverflowButton(overflowProps)}
+            </Grid>
+            {showOverflowModal && (
```

Comment:
I had it this way at one point because I was having layout issues w/ stale values, but I think I fixed that so ya I should switch back so the transition stays.